### ISSUE TITLE
Improving team games scoreboard

### ISSUE THREAD

**Issue by Lagetta** [blue-nebula/base#139]
Playing any CTF, DTF or BB needs to have more layouts to show like:
CTF: how much person has captured the flag, how much flag carries he stopped, how many returns he made, how many times he held the flag and more.
DTF: How many changed camp spots, maybe overall time how much he took changing flags.
BB: how many times he took the bomb to the base, how many times he stopped the ball, how many times he took the bomb.

There could be much more, but these things are that are really lacking in team games.

**Comment by MoonPadUSer** [comment]
I agree that there should be different statistics for different game modes, maybe let's have some base set of stats and then there will be the mode specific ones

**Comment by MoonPadUSer** [comment]
ok, I just looked into the `struct gameent` and it doesn't look like the game keeps those statistics by itself :(
